src/peft/config.py

```diff
@@ -157,7 +157,7 @@ def from_peft_type(cls, **kwargs):
 
             filtered_kwargs, unexpected_kwargs = _check_and_remove_unused_kwargs(cls, kwargs)
             if not MIN_EXPECTED_CONFIG_KEYS.issubset(set(filtered_kwargs.keys())):
-                raise TypeError(f"The config that is trying to be loaded is not a valid {cls.__name__} config.")
+                raise TypeError(f"The config that is trying to be loaded is invalid {cls.__name__} config and missing required keys: {MIN_EXPECTED_CONFIG_KEYS}.")
```

Comment:
I think the wording would not be correct after the change. Before:

> The config that is trying to be loaded is not a valid LoraConfig config

After

> The config that is trying to be loaded is invalid LoraConfig config and ...

So let's revert the wording or make it "is **an** invalid".

Moreover, let's add a line break (120 chars max).

Reply:
ok, I made the error message more concise. sounds okay?